data-integration/README.md

```diff
@@ -0,0 +1,341 @@
+# Data integration
+
+## Overview
+
+In this tutorial, you will develop a service that accepts requests to create an employee and a task, update task status, and delete a completed task. Ballerina persistence layer is used to manage data stored in a database. The service will be used to create, retrieve, update, and delete data from the database.
```

Comment:
Rather than saying just "used to manage data stored in a database", may add value if we can frame it in terms of value it adds. e.g., convenience, abstracting out the underlying db layer